Persistance/Roll-backs/gakshat468.cpp

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Comments are not needed, description is enough

Reply:
Description would be lost if not written in file

Reply:
The merged pr stays as a commit in the history, which points to the pr, the idea is that it would be helpful for the reader to go through the conversation of the review as well